container-images/scripts/build_llama_and_whisper.sh

```diff
@@ -257,21 +263,6 @@ clone_and_build_llama_cpp() {
   rm -rf llama.cpp
 }
 
-clone_and_build_ramalama() {
-  # link podman-remote to podman for use by RamaLama
```

Comment:
I'm actually using this in another PR for the libexec scripts

Reply:
You using RamaLama or using podman-remote?

Reply:
ramalama, not podman remote

Reply:
What feature of RamaLama are you needing? It is pretty small, so we can probably leave it.  But no Container support since Podman-remote is kind of chunky.

 podman-remote                      x86_64    5:5.4.1-1.fc42                      updates-testing        33.7 MiB